tests/main/views/test_direct_award.py

```diff
@@ -786,8 +786,9 @@ def test_award_contract_error_if_no_input(self):
 
         doc = html.fromstring(res.get_data(as_text=True))
         assert len(doc.xpath('//legend[contains(normalize-space(), "Select if you have awarded your contract")]')) == 1
-        assert doc.xpath('boolean(//div[@class="validation-masthead"])')
-        assert doc.xpath('count(//div[@class="validation-masthead"]/a[@class="validation-masthead-link"])') == 1
+        errors = doc.cssselect('div.govuk-error-summary a')
+        assert len(errors) == 1
+        assert errors[0].text_content() == 'Select if you have awarded your contract'
```

Comment:
Is this assertion looking at the same thing as line 788?

Reply:
No, the `legend` element in line 788 is the label for the radio input element, which includes the error message next to the form inputs, whereas line 791 is looking at the error message in the error summary.